data/data/aws/main.tf

```diff
@@ -11,6 +11,10 @@ provider "aws" {
   region = var.aws_region
 }
 
+provider "ignition" {
```

Comment:
these versions do not matter because the embedded terraform is wired to see and use embedded plugins only.